vllm/entrypoints/openai/cli_args.py

```diff
@@ -289,6 +289,19 @@ def validate_parsed_serve_args(args: argparse.Namespace):
         raise TypeError("Error: --enable-reasoning requires "
                         "--reasoning-parser")
 
+    # Ensure that --max-num-batched-tokens, --max-num-seqs, --max-model-len
+    # are passed within command on TPU.
+    from vllm.platforms import current_platform
+    if current_platform.is_tpu():
```

Comment:
We cannot put it in `platform/tpu.py` because the args are not None there. cc @NickLucche @alexm-redhat @mgoin